### ISSUE TITLE
[rule] Error with indentation

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#55]
Hey! I'm honored to open the first issue in this repo ☺️. Thanks a lot for your work in this project; it looks promising and in the good direction. I'm eager to drop htmllint in favor or LintHTML 😄.

I tested it against our current HTML source code and found something that might be a bug. Consider this code snippet:

```
<p>
  Some <strong>bold</strong> text.
</p>
```

Given the rules `"indent-style": "spaces", "indent-width": 2`, it should be considered valid HTML, shouldn't it? LintHTML complains about the incorrect indentation of the `<strong>` tag, though.

Thanks again for your time  😊

**Comment by KamiKillertO** [comment]
Thanks for reporting.
You're totally right this should not be considered as invalid, I'll fix it as soon as possible.

By the way you can totally drop htmllint as LintHTML is based from it. Normally there's no differences between them.

**Comment by acalvo** [comment]
> You're totally right this should not be considered as invalid, I'll fix it as soon as possible.

Thanks a lot 😊.

> By the way you can totally drop htmllint as LintHTML is based from it. Normally there's no differences between them.

Yesterday I did so, and LintHTML reported a few dozens of errors in our code. Almost all of them regarding indentation: most of them things that htmllint doesn't catch (yay! 🎉), some related with this bug report, and another one regarding the `attr-no-unsafe-char` rule. I didn't fill a bug report for that last one because I'm not sure whether it is or not...

The thing is that we use Angular and have some code like this:

```html
<div unrelatedAttributes [ngClass]="{
  'ok': someLongJSLogic(),
  'wrong': someLongJSLogic(),
  'warning': someLongJSLogic(),
  'info': someLongJSLogic()}">
</div>
```
It seems that LintHTML forbids the newline character inside the attributes whereas it was OK for htmllint. LintHTML might be right, but there are a few cases with too much logic for a single line. WDYT?

**Comment by KamiKillertO** [comment]
@acalvo I've just publish a new version with fixes for your problems (including the on with `attr-no-unsafe-char` 😉 )

Please open new issues if you find other problems 🙏 .
Also don't hesitate to talk about LintHTML around you, I would love to have more people using it and reporting issues/ideas... 😄

**Comment by acalvo** [comment]
Thank you, that was fast!

I run the new LintHTML version against our main HTML source code and got rid of all the errors except for one last pattern; I'll fill a bug report now.

I'm quite excited about the idea of having a proper HTML linting tool once and for all, so will be around reporting and suggesting as much as you want 😉.

Thanks again!

**Comment by vladimirov-s** [comment]
Hello. I have a similar problem at several thousands lines:
```Incorrect` indentation for `meta` beginning at L4:C5. Expected `<meta>` to be at an indentation of 8 but was found at 4.```
for the code example: 
```<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
```
while I have the settings:
```
"indent-style": "space",
"indent-width": 2,...
```
It must be Ok. What is a problem?

**Comment by KamiKillertO** [comment]
@vladimirov-s What version of LintHTML are you on?
